user_templates/moonraker.conf

```diff
@@ -34,4 +34,3 @@
 
 ##### Add your custom moonraker config customizations and overrides below this line...
 # ------------------------------------------------------------------------------------
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script